content/drone-plugins/drone-webhook/index.md

```diff
@@ -40,6 +40,21 @@ pipeline:
     urls: https://your.webhook/...
 ```
 
+Example configuration using secrets:
+(Note: tokens can be passed as passwords like this)
+
+```diff
+pipeline:
+  webhook:
+    image: plugins/webhook
++    secrets:
```

Comment:
You have to remove one space, beside that most examples are using the simple declaration:

```
    secrets: [ webhook_password ]
```